### ISSUE TITLE
Crit and Life steal items are a bit too strong

### ISSUE THREAD

**Issue by xAzta** [IroncladGames/SinsOfADarkAge#858]
Game ID: 150342

That torrace item build is just too strong, no one was able to fight him even 2 v 1. He critted and healed a lot..

In my opinion the crit chances on items are too high here, also hard cap shouldn't be 100% for chance, thats way too much.

**Comment by uncannymanny** [comment]
Recently edited.

Posting for notification's sake.

**Comment by xAzta** [comment]
Just play with several ad builds in try hero, and try this one too i listed above.

I even managed to tank 3 towers with that build :)

**Comment by megafilipe** [comment]
I agree, the game needs to fix the issue of adc's being top tier junglers.

**Comment by Zinovy** [comment]
Lifesteal and Critical Chance should not always occur together on items, I think.

We recently had a Drengar 1v4 our team and win at full health (BulletSponge was there, he knows!). He has no CC, just right click and save Leap for target switching or fake-outs. Brush contributes to the issue by providing instant cover for autoattackers to get back half their health on someone chasing, while taking half theirs. With a Khagas I have killed 3 people, fled to brush with 10 HP, then the enemy Lassidar got close I simply sidestepped the thunderbolt and got another kill in 1.3 seconds.

It really is the Speed/Lifesteal/Crit combo that does it. That game mentioned with Drengar I was Skalla and I had gone for crit items with no attention paid to attack speed - I quickly changed that and got Hiro because 71% (with Adventurer's Bow), complemented by Heartseeker were no match for their pew-pew from 2 heroes with the standard Doom Pendulum and Lifesteal shred.
